README.md

```diff
@@ -60,31 +60,49 @@ nested: configmap: {
 }
 ```
 
-Here both `namespace` and `nested.configmap` are considered as K8s manifest by cuebe.
-Whereas foo just serves during the build.
-Every manifests in a release must eventually resolves to concrete value.
-Other values can be non-concrete, as `cuebe` will not try to render them in a concrete format (YAML, JSON).
+Here both `namespace` and `nested.configmap` are considered as Manifest by Cuebe.
+Whereas foo just serves during build phase.
+Every Manifest must eventually resolves to a JSON serializable object (concrete only values).
+Other values can be non-concrete, as Cuebe will not try to render them in a concrete format (YAML, JSON).
 
-Since `cuebe` can collect any manifests in your release, it's up to you to define boundaries.
-You can see a release as a single K8s manifest,
-as a whole application (akin a [Helm](https://helm.sh) chart or a [Kustomization](https://kustomize.io/)),
-or even as a superset of multiple application (like [Helmfile](https://github.com/roboll/helmfile) or [Helmsman](https://github.com/Praqma/helmsman)).
-To help you start we designed some example on how we use `cuebe` ourself.
-If you find a new way of using `cuebe` please share it if possible.
-We're still learning and building a best practices guide.
+Since Cuebe can collect any manifests in your Build, it's up to you to define boundaries.
+You can use it to deploy single Manifest, a bunch of unrelated Manifests or use the [Instance](#instance) concept.
 
-### Attributes
+### Instance
 
+An instance is a group of [Manifests](#manifest) belonging to the same _application_.
+It's a way to group multiple manifests under the same lifecycle.
+If you're used to Helm, you can see a Cuebe Instance as a [Helm Release](https://helm.sh/docs/intro/using_helm/#three-big-concepts).
+
+A Manifest needs the `"cuebe.loft-orbital.com/instance": <instance-name>` label to be a member of the _<instance-name>_ Instance.
```

Comment:
Do we need to be more specific and state we're talking about a Kubernetes `metadata.labels` ?

Reply:
Yes, very good point